### DISCUSSION TITLE
SparkFun Thing Plus - nRF9160

### DISCUSSION THREAD

**Discussion by keenanjohnson** [micropython/micropython#10668] [nRF]
I was looking through the ports folder and noticed there wasn't an official port for the [SparkFun Thing Plus](https://www.sparkfun.com/products/17354?_ga=2.111855364.1663796225.1675547767-1676844594.1673570498) - nRF9160 board.

This is a great board and easy to buy in the US from Sparkfun. It seems like other nRF9160 boards are supported so creating an official port shouldn't be too difficult. 

I'm happy to help drive this, but could use a few pointers on how to get started. I assume taking a look at the port for the [Actinius Icarus ](https://github.com/micropython/micropython/tree/master/ports/nrf/boards/actinius_icarus)would be a good place to copy and try?

**Comment by robert-hh** [comment]
There is a good chance, that it will be easy to adapt the configuration and that the firmware will work, at least in the common feature set. I do not expect LTE being supported.

**Comment by keenanjohnson** [comment]
Thanks! Yeah, that makes sense. A custom python module would be needed to send the AT commands to configure the LTE module and everything. Thanks!
